### PR TITLE
migrate code now we know LC is always present

### LINKED ISSUE #811: Remove v1 TX and add lc to crit headers
according to rfc016: https://nuts-foundation.gitbook.io/drafts/rfc/rfc016-lamport-clock

### LINKED ISSUE #1122: remove calculated LC value
Requires V1 protocol removal and development network reset

### PR DESCRIPTION
**Pull request by woutslakhorst**
Closes #811
Closes #1122

### CODE REVIEW

File: network/dag/bbolt_tree.go [hunk 1]
```diff
@@ -115,32 +114,6 @@ func (store *bboltTree) dagObserver(ctx context.Context, transaction Transaction
 	return nil
 }
 
-// migrate builds a tree by walking over the dag and adding all Transaction references to the tree without checking for validity.
-// The tree is stored on disk once it is in sync with the dag.
-func (store *bboltTree) migrate(ctx context.Context, state State) error {
```
Comment (gerardsn) [inline network/dag/bbolt_tree.go:120]: This should be replaced with something else to allow rebuilding the tree

**Review by gerardsn — APPROVED** [review]
LGTM

File: network/dag/bbolt_tree.go (hunk 1 above)
Comment (woutslakhorst) [reply network/dag/bbolt_tree.go:120]: I'll create an issue for that

Comment (gerardsn) [reply network/dag/bbolt_tree.go:120]: True. Maybe not needed then...

**Pull request merged**